libs/ui-toolkit/src/components/indicator/indicator.tsx

```diff
@@ -0,0 +1,11 @@
+import classNames from 'classnames';
+import type { Variant } from '../../utils/intent';
+import { getVariantBackground } from '../../utils/intent';
+
+export const Indicator = ({ variant }: { variant?: Variant }) => {
+  const names = classNames(
+    'inline-block w-8 h-8 mb-[0.15rem] mr-8 rounded',
```

Comment:
I don't like this custom spacing values. It's better to use to use mb-2. It's 0.125 rem which gives 2px. 0.15rem is 2.4px which will be rounded to 2px